tensorrt_llm/serve/openai_server.py

```diff
@@ -229,6 +249,7 @@ async def chat_stream_generator(
                 pp_results = res.outputs[0]._postprocess_result if self.postproc_worker_enabled else post_processor(res, args)
                 for pp_res in pp_results:
                     yield pp_res
+            logger.info(f"generation completed for request {raw_request.state.request_id}")
```

Comment:
Any reason we don't print perf_counter here as well?

Reply:
I realized that the perf_counter might be superfluous - we already have log time stamps, but i can follow up by making these consistent (both have perf counter or neither)